src/test/java/net/testudobank/tests/MvcControllerIntegTest.java

```diff
@@ -239,6 +239,69 @@ public void testWithdrawTriggersOverdraft() throws SQLException, ScriptException
     MvcControllerIntegTestHelpers.checkTransactionLog(customer1TransactionLog, timeWhenWithdrawRequestSent, CUSTOMER1_ID, MvcController.TRANSACTION_HISTORY_WITHDRAW_ACTION, CUSTOMER1_AMOUNT_TO_WITHDRAW_IN_PENNIES);
   }
 
+/**
+   * Verifies the case where a customer withdraws an amount that causes the
+   * customer to enter overdraft, and the overdraft balance with interest applied
+   * doesn't exceed the overdraft limit.
```

Comment:
Delete this first paragraph because it conflicts with the 2 paragraphs you have written below.

Reply:
Thanks, I did not notice that this first paragraph was still here. I have removed it now.